packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -143,88 +156,94 @@ String addServiceExtensions() {
                     });
                   }
                   additionalJson['constraints'] = constraintsProperty;
+                } catch (e) {
+                  // Not laid out yet.
                 }
                 if (renderObject is RenderBox) {
                   additionalJson['size'] = <String, Object>{
                     'width': renderObject.size.width.toString(),
                     'height': renderObject.size.height.toString(),
                   };
 
-                  final ParentData parentData = renderObject.parentData;
+                  final ParentData parentData =
+                      renderObject.parentData as ParentData;
```

Comment:
`final ParentData` and `as ParentData` don't these do the same?  Only need one?

Reply:
you are right. these types don't need to be on the LHS but they do need to be on the RHS. This code matches flutter style of putting types everywhere so it is easier to merge into package:flutter.